tests/console/rmt/rmt_migration.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: rmt-server
+# Summary: Add rmt configuration test and basic configuration via
+#    rmt-wizard, enable repo, check repo at base system, then upgrade
+#    to latest one. Check rmt still can work and repo still exsites.
+# Maintainer: Yutao wang <yuwang@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use repo_tools;
+
+sub run {
+    select_console 'root-console';
+
+    if (check_var('VERSION', get_required_var('ORIGIN_SYSTEM_VERSION'))) {
+        rmt_wizard();
+        # Sync from SCC
+        rmt_sync;
+        # Enable one product
+        assert_script_run("rmt-cli product enable sle-module-live-patching/15/x86_64");
```

Comment:
You can use a variable for the hard code live-patching module path, another solution is to set this in SETTING so that we can change the module easily.

Reply:
This is just test it can be enable. Don't need set it to the setting.
Like rmt_feature, We have discussed before.